code/modules/paperwork/paper.dm

```diff
@@ -345,6 +361,33 @@
 			else
 				to_chat(user, SPAN_DANGER("You must hold \the [P] steady to burn \the [src]."))
 
+/obj/item/paper/verb/seal_paper()
+	set name = "Seal paper"
+	set category = "Object"
+	set src in usr
+
+	var/confirm = alert(usr, "Are you sure you wish to seal this document?\n\nNote: This will prevent the edit of any fields on the paper,\nexcluding the end of the page.", "Confirm", "Yes", "No")
+	if(confirm != "Yes")
+		return FALSE
```

Comment:
```suggestion
	if(tgui_alert(usr, "Are you sure you wish to seal this document? Note: This will prevent the edit of any fields on the paper, excluding the end of the page.", "Confirm", list("Yes", "No"), 20 SECONDS) != "Yes")
		return FALSE
```